### PR TITLE
bump deps

### PR DESCRIPTION
**Pull request by acim**

### CODE REVIEW

File: impl/Cargo.toml [hunk 1]
```diff
@@ -15,12 +15,12 @@ edition = "2018"
 proc-macro = true
 
 [dependencies]
-rust-embed-utils = { version = "7.1.0", path = "../utils"}
+rust-embed-utils = { version = "7.2.0", path = "../utils"}
```
Comment (AzureMarker) [inline impl/Cargo.toml:18]: If this doesn't need to be bumped, then don't bump it. Bumping it only limits the possible dependency versions users can use.

File: impl/Cargo.toml [hunk 2]
```diff
@@ -15,12 +15,12 @@ edition = "2018"
...
 
 syn = { version = "1", default-features = false, features = ["derive", "parsing", "proc-macro"] }
 quote = "1"
 proc-macro2 = "1"
-walkdir = "2.3.1"
+walkdir = "2.3.2"
```
Comment (AzureMarker) [inline impl/Cargo.toml:23]: Same with this dependency. Users will automatically use the latest compatible version, so we don't need to force them.

**Comment by acim** [comment]
@AzureMarker, thank you for the suggestions, this is my first Rust PR.

**Comment by AzureMarker** [comment]
Thanks for the PR!

**Pull request merged**